src/main/java/com/monumental/controllers/MonumentController.java

```diff
@@ -51,17 +61,33 @@ public Monument createMonument(@RequestBody CreateMonumentRequest monumentReques
     }
 
     /**
-     * Get a Monument with the specified ID, if it exists
+     * Get a Monument with the specified ID, if it exists and is active or inactive depending on onlyActive
      * @param id - ID of the Monument to get
      * @param cascade - If true, loads all of the lazy-loaded collections associated with the Monument
+     * @param onlyActive - If true, a 404 will be returned if the specified Monument is inactive. If false, the monument
+     *                 will be returned regardless of whether it's active or inactive.
+     *                 If this is false then the user must be a partner or researcher to view
      * @return Monument - The Monument with the specified ID, if it exists
-     * @throws ResourceNotFoundException - If a Monument with the specified ID does not exist
+     * @throws ResourceNotFoundException - If a Monument with the specified ID does not exist or onlyActive is true and isActive is false
+     * @throws AccessDeniedException - If trying to get an inactive monument without being a partner or researcher
```

Comment:
Same here: `If trying to get an inactive monument without being a partner or above`.